### ISSUE TITLE
DarkFace Train/Val Split

### ISSUE THREAD

**Issue by yanqinJiang** [daooshee/HLA-Face-Code#5]
Hi, thanks for your great work! I want to know how you split DarkFace in to train&val set. Specifically, which 500 images are selected as val images? Moreover, are models in Table. 1 in the paper evaluated on val set or test set? Looking forward to your reply, 3Q!

**Comment by daooshee** [comment]
Hi, this is my split for DARK FACE training and validation:
Train: [mf_dsfd_dark_face_val_500.txt](https://github.com/daooshee/HLA-Face-Code/files/6867825/mf_dsfd_dark_face_val_500.txt)
Val: [mf_dsfd_dark_face_train_5500.txt](https://github.com/daooshee/HLA-Face-Code/files/6867827/mf_dsfd_dark_face_train_5500.txt)
Recently, I realized that since the model cannot use DARK FACE annotations for validation, splitting 500 images for validation is actually meaningless. Therefore I suggest using [mf_dsfd_dark_face_train.txt](https://github.com/daooshee/HLA-Face-Code/blob/main/train_code/dataset/mf_dsfd_dark_face_train.txt) for both training and validation.

The models in Table 1 are evaluated on the test set.

If you have any other questions, please feel free to ask me~

**Comment by yanqinJiang** [comment]
thanks a lot!

**Comment by yanqinJiang** [comment]
Sorry to bother you again, but I have a few more questions. You said "splitting 500 images for validation is actually meaningless", does that mean you didn't use validation set in your paper? Are ablations in Table 3 also done on test set? Do you use images for both train and val when training hla-face? Looking forward to your reply and thanks again!

**Comment by daooshee** [comment]
> You said "splitting 500 images for validation is actually meaningless", does that mean you didn't use validation set in your paper ?

No, I did use the train/val splitting in the paper.

> Are ablations in Table 3 also done on test set?

Yes, it is done on the test set. The mAP of full version (44.4) in Table 3 is consistent with the mAP of Ours (44.4) in Table 1.

> Do you use images for both train and val when training hla-face?

I only use train images when training the model.

**Comment by yanqinJiang** [comment]
Thanks for your patient reply! That helps a lot!

**Comment by zhebawomenjiuyaochangshenle** [comment]
Hey, why do some images in your mf_dsfd_dark_face_val_500.txt and MF _ dsfd _ dark _ face _ train _ 5500.txt have a large number of annotations, which is different from the officially downloaded ones?
